src/plone/versioncheck/formatter.py

```diff
@@ -202,7 +202,8 @@ def human(
                 version['version'] +
                 dots(version['version'], pkgsinfo['ver_maxlen']) +
                 ' ' + color_by_state(version['state']) +
-                version['state'][0] + ' ' + version['description']
+                version['state'][0] if version['state'] else '' +
```

Comment:
As an alternative, we could just do ``version['state']`` without the list access. AFAI can see, version['state'] is filled with one-letter states anyways in:
https://github.com/plone/plone.versioncheck/blob/master/src/plone/versioncheck/formatter.py#L21